src/TestInfrastructure/VsObjectModelMocks/ProjectMock.cs

```diff
@@ -310,10 +310,12 @@ int IVsProject.AddItem(uint itemidLoc, VSADDITEMOPERATION dwAddItemOperation, st
 
         int IVsBuildPropertyStorage.GetPropertyValue(string pszPropName, string pszConfigName, uint storage, out string pbstrPropValue)
         {
+            string propertyKey = GetInternalBuildPropertyKey(pszPropName, pszConfigName);
```

Comment:
The test framework now needs to handle different properties for different configurations. To keep it simple, the _buildProperties_ dictionary key is now a composite key of _property name _ + _configuration_.